test/Config.js

```diff
@@ -85,14 +85,24 @@ test('toConfig with values', t => {
     .output
       .path('build')
       .end()
+    .mode("development")
```

Comment:
Can you replace all the double quotes in here with single quotes to match the style? I need to go in and add better linting tools some day. 😄